### ISSUE TITLE
GQL - unable to query grouped fields in submissions

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#368]
**Description**
We have been running into an error trying to query submissions with grouped fields, looks like they might be typed wrong

As shown in the following screenshots, we are unable to access `petName` inside `generalDetails` because it's expected to be a string, yet querying `generalDetails` as a string also throws an error

![Screen Shot 2021-02-25 at 2 50 33 pm](https://user-images.githubusercontent.com/21274925/109104736-748d4f00-777c-11eb-8f65-6bfd04f9109d.png)

![Screen Shot 2021-02-25 at 2 50 44 pm](https://user-images.githubusercontent.com/21274925/109104750-7d7e2080-777c-11eb-8933-828ff2c74de4.png)

Sorry to keep coming back with issues 😅

**Steps to reproduce**
1. Create a form with grouped fields
2. Make a submission
3. Try to query grouped fields in the submission

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.19.1
- Craft version: 3.6.7
- Multi-site: No

**Comment by engram-design** [comment]
Yep, working on it! See https://github.com/verbb/formie/issues/299

**Comment by d-karstens** [comment]
Thank you!

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Thank you!

**Comment by engram-design** [comment]
Fixed in [1.3.22](https://github.com/verbb/formie/releases/tag/1.3.22)
